src/lib/api-client.js

```diff
@@ -152,6 +152,19 @@ export function getOverdueItems() {
     return get('item/overdue');
 }
 
+export function saveItem(itemAddress) {
+    return post('item/save', {
+        itemAddress
```

Comment:
It does not. There can only be one student per item address. We need the student id for models because each model address could be checked out by multiple students.